internal/state/store.go

```diff
@@ -31,10 +31,11 @@ const (
 // TODO(thane): Move these and the ones in internal/store/store.go to their own package.
 const (
 	// prefixes are unique across all tm db's
-	prefixValidators      = int64(5)
-	prefixConsensusParams = int64(6)
-	prefixABCIResponses   = int64(7)
-	prefixState           = int64(8)
+	prefixValidators             = int64(5)
+	prefixConsensusParams        = int64(6)
+	prefixABCIResponses          = int64(7) // deprecated in v0.36
+	prefixState                  = int64(8)
+	prefixFinalizeBlockResponses = int64(9)
```

Comment:
I think 9 may already be used here: https://github.com/tendermint/tendermint/blob/master/internal/evidence/pool.go#L27

Reply:
It is somewhat likely that we would never actually attempt to combine these prefixes in the same database anyway—so I think the risk in practice is pretty low. Still, it's probably safest to keep them (completely) isolated.

Reply:
(I only know about this because in the key migration script, we process all the prefix types at once, even though each table is handled separately, and it bit me there—here we are not so exposed because the keys should already be migrated)

Reply:
I did a bit of digging. There are actually all these files using prefixes:
* internal/evidence/pool.go
* internal/p2p/peermanager.go
* internal/state/store.go
* internal/store/store.go
* light/store/db/db.go
* scripts/scmigrate/migrate.go

I'll rearrange them

Reply:
scripts/keymigrate/migrate.go uses them too.